subsys/bluetooth/host/classic/Kconfig

```diff
@@ -26,6 +26,12 @@ config BT_MAX_SCO_CONN
 	  Maximum number of simultaneous Bluetooth synchronous connections
 	  supported. The minimum (and default) number is 1.
 
+config BT_L2CAP_CLS
+	bool "Bluetooth L2CAP connectionless data channel [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  This option enables Bluetooth L2CAP connectionless data channel
```

Comment:
What exactly do you need this feature for? I know it exists in the core spec, but I don't remember any real use cases for it. Let's not add things unless they're needed by some profile for which you have concrete product requirements.